src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime()
+        if(!image) {
+            return
+        }
+        try {
+            retry(3) {
+                def dd = e.git.url
+                echo "YZAINEE: ${dd}"
+                def response = bayesianAnalysis url: 'https://bayesian-link', gitUrl: e.git.url, ecosystem: image
```

Comment:
doesn't analytics need git sha1 ?

Reply:
This will be removed. This was just to test if i get the giturl as mentioned by @hrishin  in the comment above. As of now, its not working. I get null pointed when i try to get it from e.git.url